docs/topics/test-system.md

```diff
@@ -1,72 +1,126 @@
 # Test-system
 
+The test systems are located on a CloudVPS machine at `wikibase-product-testing-2022.wikidata-dev.eqiad1.wikimedia.cloud`.
 
-## Configuration
+You can find all code for running test test systems in the `/opt/test-systems` directory.
+This directory is owned by the `mediawiki` user.
+Each test system is a copy of the `example` docker-compose setup, with customized env vars and settings.
 
-The test-system is currently using a non versioned copy of the [example](../../example/README.md) folder deployed to `wb-product-testing.wikidata-dev.eqiad.wmflabs`.
+Two test systems are maintained, one for the current "latest" major release, and one for the "previous" major release.
 
-Files are located under the `/opt/` folder and for keeping some kind of papertrail a copy of the version to be tested is usually created as `/opt/wmde.1`, `/opt/wmde.2` etc.
+- Latest release (internal ports 83**)
+  - https://wikibase-product-testing.wmcloud.org (8380 internal)
+  - https://wikibase-query-testing.wmcloud.org (8381 internal)
+  - https://wikibase-qs-testing.wmcloud.org (8382 internal)
+- Previous release (internal ports 82**)
+  - https://wikibase-product-testing-previous.wmcloud.org (8280 internal)
+  - https://wikibase-query-testing-previous.wmcloud.org (8281 internal)
+  - https://wikibase-qs-testing-previous.wmcloud.org (8282 internal)
```

Comment:
Don't seem to be able to initiate oauth with this

Reply:
something with 

http://wikibase-product-testing-previous.wmflabs.org:8280/w/index.php?title=Special:OAuth/authorize&oauth_token=db04ab57974ab6233b3c4fe26bb2f6dd&oauth_consumer_key=55121eeebe592797244734b5ae274e2d&callback=https%3A%2F%2Fwikibase-qs-testing-previous.wmflabs.org%3A8282%2Fapi.php

Reply:
Resolved with Resolved with https://github.com/wmde/wikibase-release-pipeline/pull/315/commits/f9dadd1b0e132a19cef942e79721e8231e34eb04